sonar-plugin-api/src/main/java/org/sonar/api/batch/fs/InputFile.java

```diff
@@ -91,7 +97,9 @@
    * @see #contents()
    * @see #inputStream()
    * @since 5.1
+   * @deprecated since 6.5 use {@link #inputStream()}, {@link #filename()} or {@link #uri()}
    */
+  @Deprecated
   @Override
   Path path();
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)